verification/src/http_server/handlers/status.rs

```diff
@@ -1,5 +1,10 @@
-use actix_web::{HttpResponse, Responder};
+use actix_web::Responder;
+use paperclip::actix::{api_v2_operation, web::HttpResponse};
 
+#[api_v2_operation(
+    summary = "Health check",
+    description = "Returns a 200 response if the server is healthy"
+)]
```

Comment:
At the moment `paperclip` can't deduce `impl Repsonder`, `HttpResponse`, etc.
It becomes a problem in such simple cases like that: `paperclip` can not say that return code is always `200 OK`.
If we take a look at the [source code](https://github.com/paperclip-rs/paperclip/blob/master/macros/src/actix.rs#L169), we can only specify `summary` and `description`

Reply:
Used `NoContent 204` status code for that, I guess that's ok